.github/workflows/icu_ci.yml

```diff
@@ -440,3 +440,32 @@ jobs:
           echo Did you forget to include the changed data files in this PR?
           exit 1
         fi
+
+  # Build and run ICU4C samples
+  icu4c-test-samples:
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/checkout@v2
+
+      - name: ICU4C configure and build
+        env:
+          PREFIX: icu4c/source
+        run: |
+          # Perform an in-source build of icu4c
+          cd icu4c/source
+          ./runConfigureICU Linux
+          make -j2;
+          # Reference the paths in the in new build
+          export PATH=$PATH:$PREFIX/bin
```

Comment:
This and the following statement are intended for the next step ('Build and run samples'). As such they should be in the next step, perhaps in an 'env:' clause.